### DISCUSSION TITLE
Open Issue about lill fix for Tree or waiting till new major version is released?

### DISCUSSION THREAD

**Discussion by Mad4emno** [ghiscoding/Angular-Slickgrid#810] [General]
I found a lill problem in the Tree if u wanna let own data from a db show up automatically. 
Have already a fix for my own, but I guess it could be fixed in the lib too.
Shall I open an issue or wait for the new major version and then look if this problem is still there?

**Comment by ghiscoding** [comment]
it depends what that is, you can explain a bit more about the problem and see what it is

**Comment by ghiscoding** [comment]
@Mad4emno The new `major 3.x` is out, so what was your fix about Tree Data?
